### PR TITLE
15st1 [2905][ADD] account_invoice_send_wizard_reply_to

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[2905](https://www.quartile.co/web#view_type=form&model=project.task&id=2905&active_id=2905&menu_id=)

### CODE REVIEW

File: account_invoice_send_mail_reply_to/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This module does the following:
+
+- Respects the 'reply-to' setting of the email template in account invoice send mail unless the value was otherwise specified.
```
Comment (yostashiro) [inline account_invoice_send_mail_reply_to/readme/DESCRIPTION.rst]:
```suggestion
- Adds reply_to field to account.account_invoice_send_wizard_form.

This adjustment is necessary for the custom module mail_compose_message_reply_to to work as expected
when an invoice is manually sent via the wizard.
```

The current description is not technically about this module.

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Review by yostashiro — CHANGES_REQUESTED** [review]
Please rename the module to account_invoice_send_wizard_reply_to.

Please put `account_invoice_send_views.xml` in `wizards` folder.

**Comment by kanda999** [comment]
@AungKoKoLin1997 
This module is not working as expected.
・I have tested and reply_to set is not used.
・I have confirmed that both account_invoice_send_reply_to and mail_compose_message_reply_to installed.

Because DB is made in JP??

**Comment by AungKoKoLin1997** [comment]
> @AungKoKoLin1997 This module is not working as expected. ・I have tested and reply_to set is not used. ・I have confirmed that both account_invoice_send_reply_to and mail_compose_message_reply_to installed.
@kanda999 . Please watch this demo video https://drive.google.com/file/d/1JK84BSbSMSvuZpU4onSbeJGQZpTsKiXD/view?usp=share_link. I think currently you can only watch this only with 360p because google is processing this video. I suggest you to download and watch. Please let me know if something is missing.
> Because DB is made in JP??
No. My db is also created with japan.

**Review by kanda999 — APPROVED** [review]
Function: review: Worked as expected in mi715test11 environment.

But this module did not work in my development environment.

File: account_invoice_send_wizard_reply_to/README.rst [hunk 2]
```diff
@@ -14,14 +14,17 @@ Account Invoice Send Reply To
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-qrtl%2Fmi7--custom-lightgray.png?logo=github
-    :target: https://github.com/qrtl/mi7-custom/tree/17.0/account_invoice_send_mail_reply_to
+    :target: https://github.com/qrtl/mi7-custom/tree/15st1/account_invoice_send_wizard_reply_to
```
Comment (yostashiro) [inline account_invoice_send_wizard_reply_to/README.rst]: Regenerate readme for 15.0.

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Comment by kanda999** [comment]
This PR is closed because I followed up with https://github.com/qrtl/mi7-custom/pull/218

**Pull request closed without merging**